src/components/Person.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is&nbsp;
+      {person.name}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        I am&nbsp;
+        {person.age}
+      </p>
+    )}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        My &nbsp;
+        {person.sex === 'f' ? 'wife' : 'husbend'}
```

Comment:
```suggestion
        {person.sex === 'f' ? 'husband' : 'wife'}
```
Тут має бути навпаки. Також помилка в слові husband